### DISCUSSION TITLE
JPA Auditing  란?

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#24] [Q&A] [answered]
JPA Auditing  란?

**Comment by tjdud22** [comment, marked as answer]
JPA에서 시간을 자동으로 값을 넣어주는 기능입니다. Entity를 영속성 컨텍스트에 저장하거나 조회를 수행한 후에 update를 하는 경우 매번 시간 데이터를 입력하여 주어야 하는데, JPA Auditing을 이용하면 자동으로 시간을 매핑하여 데이터베이스의 테이블에 넣어주게 됩니다.
